app/routes/surveys/components/SurveyEditor.js

```diff
@@ -0,0 +1,134 @@
+// @flow
+
+import styles from './surveys.css';
+import React, { Component } from 'react';
+import { DetailNavigation, ListNavigation } from '../utils.js';
+import { Field } from 'redux-form';
+import Button from 'app/components/Button';
+import {
+  TextInput,
+  RadioButton,
+  SelectInput,
+  RadioButtonGroup,
+  DatePicker
+} from 'app/components/Form';
+import { createValidator, required } from 'app/utils/validation';
+import { reduxForm } from 'redux-form';
+import type { SurveyEntity } from 'app/reducers/surveys';
+import Content from 'app/components/Layout/Content';
+import type { FieldProps } from 'redux-form';
+
+type Props = FieldProps & {
+  survey: SurveyEntity,
+  autoFocus: any,
+  submitFunction: (SurveyEntity, ?number) => Promise<*>,
+  deleteSurvey: number => Promise<*>,
+  push: string => void
+};
+
+class SurveyEditor extends Component<Props> {
+  onSubmit = (formContent: Object) => {
+    const { survey, submitFunction, push } = this.props;
+    return submitFunction({
+      ...formContent,
+      event: formContent.event && Number(formContent.event.value),
+      surveyId: survey && survey.id
+    }).then(result => {
+      const id = survey ? survey.id : result.payload.result;
+      push(`/surveys/${String(id)}`);
+    });
+  };
+
+  render() {
+    const {
+      survey,
+      submitting,
+      autoFocus,
+      handleSubmit,
+      deleteSurvey
+    } = this.props;
+
+    const titleField = (
+      <Field
+        placeholder="Tittel"
+        label=" "
+        autoFocus={autoFocus}
+        name="title"
+        component={TextInput.Field}
+        className={styles.editTitle}
+      />
+    );
+
+    return (
+      <Content>
+        <div className={styles.detail}>
```

Comment:
Isn't it possible to add `styles.detail` to the `Content`-tag, to avoid two divs?